src/conn/conn_prefetch.c

```diff
@@ -190,6 +193,8 @@ __wt_conn_prefetch_queue_push(WT_SESSION_IMPL *session, WT_REF *ref)
         WT_ERR(EBUSY);
     }
 
+    /* /* We should never add a ref that is already in the prefetch queue. */
+    WT_ASSERT(session, !F_ISSET_ATOMIC_8(ref, WT_REF_FLAG_PREFETCH));
```

Comment:
We already check for this in the caller of the `__wt_conn_prefetch_queue_push` function. We should remove the check from one of the places.